crux-agent/examples/context_management_example.py

```
@@ -0,0 +1,205 @@
+1. Set custom context limits
+2. Monitor context usage
+3. Test rolling window behavior with long iterations
+"""
+
+import os
+import sys
+
+# Add the project root to Python path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
+
+from app.core.providers.lmstudio import LMStudioProvider
```

Ruff (0.12.2)

`app.core.providers.lmstudio.LMStudioProvider` imported but unused

Remove unused import: `app.core.providers.lmstudio.LMStudioProvider`

(F401)